README.md

```diff
@@ -6,54 +6,55 @@ A tool used to validate firefox experiments
-- ```test_experiment_remains_disabled_after_user_disables_it```: Disable experiment, restart Firefox to make sure it stays disabled.
-- ```test_experiment_sends_correct_telemetry```: Make sure telemetry is sent and recieved properly.
-- ```test_experiment_does_not_stop_update```: Experiment should not block firefox updates.
+Here is an example: ```docker compose run klaatu tox -e bdd-tests -- --experiment-branch control --variables tests/fixtures/test_experiment.json --private-browsing-enabled```. By default this will build and run the tests on Firefox Release.
 
+There are 3 versions of Klaatu that have each major Firefox type. 
+1. `mozilla/klaatu:firefox-release` : Firefox Release
+2. `mozilla/klaatu:firefox-beta`: Firefox Beta
+3. `mozilla/klaatu:firefox-nightly`: Firefox nightly
 
 ## Prerequisites
 
 You should have docker and git installed.
 
-## How to use
+## Building locally
 
 1. Clone repository
-2. Fill out this JSON file, name it `variables.json` and place it in the root directory:
-
+2. Copy the JSON of your experiment, into a file named `experiment.json`. The JSON can be found in 2 ways. On production:
+```
+https://experimenter.services.mozilla.com/api/v6/experiments/{EXPERIMENT-SLUG-HERE}
 ```
-    {
-        "recipeId":
-        "slug":
-        "userFacingName":
-        "userFacingDescription":
-        "branch":
-        "active": true,
-        "addonId":
-        "addonUrl": "https://example.com/{YOUR ADDON ID HERE}@mozilla.org-signed.xpi",
-        "addonVersion":
-        "extensionApiId":
-        "extensionHash": "badhash",
-        "hashAlgorithm": "sha256",
-        "studyEndDate": null
-    }
+Or Stage:
 ```
-Add the path using the ```--variables``` option. ```--variables={PATH/TO/variables.json}```
+https://stage.experimenter.nonprod.dataops.mozgcp.net/api/v6/experiments/{EXPERIMENT-SLUG-HERE}
+```
+Place this file in the somewhere within the working directory.
+
+3. Add the path using the ```--variables``` option. Ex: ```--variables={PATH/TO/experiment.json}```
+4. Add the branch you want to test with the `--experiment-branch` option. Ex: `--experiment-branch control`
+
+5. Build docker image with command `FIREFOX_VERSION="-nightly" docker compose -f docker-compose.yml up -d --build` in the projects root directory. The `FIREFOX_VERSION` env variable can either be `-nightly` or `-beta`. Leave it blank to build for release: `FIREFOX_VERSION=""`.
+6. Run tests with docker, example: ```docker compose run klaatu tox -e bdd-tests -- --experiment-branch={BRANCH YOU WANT TO TEST} --variables={PATH/TO/variables.json}```
+
+## Running on Windows
+
+The file `docker-compose-windows.yml` contains a windows docker image. This setup is much more involved but I've provided some scripts to help with this.
+
+1. The method of interacting with the docker image is through RDP. I suggest using xrdp[1] for this. If you're on windows but using WSL, you can use the build in RDP client.
```

Comment:
I don't think this is a new issue, but what is the `[1]` footnote in `xrdp[1]`?
I'm not seeing any other instances of `[1]` with links or references to xrdp.

Reply:
Ah crap, good catch, forgot to add that